src/MoBi.R/Services/MoBiRIntegrationService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MoBi.Core.Domain.Model;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.ParameterIdentifications;
+using ISerializationTask = MoBi.Presentation.Tasks.ISerializationTask;
+
+namespace MoBi.R.Services
+{
+   public interface IMoBiRIntegrationService
+   {
+      IndividualBuildingBlock CreateIndividual(string name);
+      void SetParameterValue(ParameterValuesBuildingBlock pvBB, string parameterName, double newValue);
+   }
+
+   public class MoBiRIntegrationService : IMoBiRIntegrationService
+   {
+      public IndividualBuildingBlock CreateIndividual(string name)
+      {
+         return new IndividualBuildingBlock { Name = name };
+      }
```

Comment:
Most of the time when you're doing something like this for `IndividualBuildingBlock` there should be something like it for `ExpressionProfileBuildingBlock` as well.

Also, can you verify what's being asked for in this use case? This is an empty building block, and in MoBi an `IndividualBuildingBlock` can't be modified (except the values of individual parameters). So is the user looking for us to create a new PKSim `IndividualBuildingBlock`?